packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -142,13 +142,14 @@ def test_octarine_srv(cluster, timeout=30):
 
 def test_pkgpanda_api(cluster):
 
-    def get_and_validate_package_ids(node, uri):
-        r = cluster.get(node=node, path=uri)
+    def get_and_validate_package_ids(node, path):
+        node_url = cluster.get_node_url(node)
```

Comment:
the two calls here are a considerable API degredation from previously. Maybe cluster() needs to override / inject into the ClusterApi() a wrapper which makes it so that it will take either "host_url", _or_ if given the keyword parameter `node=` it will calculate host_url from that (both given or neither given being an error).